src/cms/forms/pages/page_form.py

```diff
@@ -76,12 +76,30 @@ class MirrorPageField(forms.ModelChoiceField):
 
     # pylint: disable=arguments-differ
     def label_from_instance(self, page):
-        return " -> ".join(
+        label = " 🡒 ".join(
             [
                 page_iter.best_language_title()
                 for page_iter in page.get_ancestors(include_self=True)
             ]
         )
+        if page.archived:
+            label += " (⚠ " + _("Archived") + ")"
+        return label
```

Comment:
Seems like some kind of encoding error
### Before
![before](https://user-images.githubusercontent.com/47010475/105017874-98a29580-5a44-11eb-89a9-6399c2eddcc2.png)
### Afterward
![afterwards](https://user-images.githubusercontent.com/47010475/105017881-9c361c80-5a44-11eb-9d51-839401ff515c.png)

Reply:
Interesting, in my browser it works:tm:
I tried a different approach now, can you check whether this works on your system? Thanks!

Reply:
Yes, now the arrow is rendered correctly:tada:. I'm on firefox, so might have been a firefox issue.